### PR TITLE
specific file name with some info

### PR DESCRIPTION
**Pull request by yrq110**

### CODE REVIEW

File: src/components/App.vue [hunk 1]
```diff
@@ -102,8 +103,14 @@ export default {
       if (!url || this.downloading) return
 
       this.downloading = true
-      const res = await this.fetchImage(url)
-      download(res.url)
+      const file_name = `${res.author}-${res.media_id}.jpg`
+      const dl = await axios({
```
Comment (egoist) [inline src/components/App.vue]: no need to use axios, we can directly `download(url, filename)`

File: src/components/App.vue [hunk 2]
```diff
@@ -78,7 +78,8 @@ export default {
         return {
           url: getOriginalImage(res.thumbnail_url),
           author: res.author_name,
-          desc: res.title
+          desc: res.title,
+          media_id: res.media_id
```
Comment (egoist) [inline src/components/App.vue]: maybe set <code>filename: `${res.author}-${res.media_id}`</code> here directly.

File: src/components/App.vue (hunk 1 above)
Comment (yrq110) [inline src/components/App.vue]: 1. I tried directly `download(url, filename)` but the downloaded file is not valid image.  
After see the [downloadjs](https://github.com/rndme/download) doc, i found get blob response data and then set filename and mime type worked.
2. delete `const res = await this.fetchImage(url)` by mistake

File: src/components/App.vue (hunk 2 above)
Comment (yrq110) [inline src/components/App.vue]: ok, it seems better

**Comment by egoist** [comment]
Thanks, the new version has been deployed.

**Pull request merged**